include/fxcg/display.h

```diff
@@ -87,6 +87,11 @@ void Bdisp_ShapeBase3XVRAM( void*shape );
 void Bdisp_ShapeBase( unsigned char*work, struct display_shape *shape, int color, int line_width, int zero1, int zero2 );
 void Bdisp_ShapeToVRAM16C( void*, int color );
 void Bdisp_ShapeToDD( void*shape, int color );
+
+void Bdisp_Rectangle( int x1, int y1, int x2, int y2, char color );
```

Comment:
Can you add comments to each of these describing what they do? Just what you've put in the PR description would be fine.

Reply:
I was planning on adding them to WikiPrizm, but I can do this too.